chapters/en/slides/module1/module1_31.Rmd

```diff
@@ -157,7 +154,7 @@ chart0 = alt.Chart(cereal, width=500, height=300).mark_bar().encode(
     x='mfr',
     y='count()'
 )
-chart1
+chart0
```

Comment:
Since this is the second plot, can you rename this to `chart1` and the one above it to `chart0` ? I actually fixed this on the master butI don't think you pulled the changes!